packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -846,9 +846,14 @@ abstract class SliderComponentShape {
 ///  * [SliderTrackShape] Base component for creating other custom track
 ///    shapes.
 class RectangularSliderTrackShape extends SliderTrackShape {
-  /// Abstract const constructor. This constructor enables subclasses to provide
-  /// const constructors so that they can be used in const expressions.
-  const RectangularSliderTrackShape();
+  /// Create a slider track that draws 2 rectangles.
+  const RectangularSliderTrackShape({this.disabledThumbGap = 2.0});
```

Comment:
const RectangularSliderTrackShape({ this.disabledThumbGap = 2.0 });

Reply:
Done